api/src/api/users.ts

```diff
@@ -80,11 +80,13 @@ router.put(
       return;
     }
 
-    const updatedUser = await User.findByIdAndUpdate(
-      req.params.userId,
-      req.body,
-      { new: true, runValidators: true },
-    );
+    const user = req.body;
+    Object.keys(user).forEach((key) => user[key] === null && delete user[key]);
```

Comment:
Whoa :0 what is this for, exactly?

Reply:
It just removes the fields we don't want to update